src/index.test.js

```diff
@@ -53,6 +61,13 @@ describe('metrics agent', () => {
     expect(typeof agent).toEqual('function');
   });
 
+  it('should create a new dnsPromise on lib config', () => {
+    process.env.DNS_ON_LIB = true;
+    const agent = IOpipe();
+    expect(agent.dnsPromise).toBeTruthy();
+    process.env.DNS_ON_LIB = false;
```

Comment:
because environment variables are always strings, you have to delete an environment variable for it to be falsey